elastic/kibana
# [Infrastructure UI] Propagate Kibana version to hosts view feedback form

closes: [#1053](https://github.com/elastic/obs-infraobs-team/issues/1053)
## Summary

This PR changes the Hosts View feedback button to pass also the current Kibana version. 

<img width="716" alt="image" src="https://github.com/elastic/kibana/assets/2767137/9089d2c3-590c-46c2-89ce-c6bd7bd42a6b">

The URL is no longer a shortened one, and that's because it wasn't forwarding query parameters.


### How to test
- Start a local Kibana
- Navigate to `Infrastructure > Hosts`
- Click on "Tell us what you think" button
- On the forms, the question "What version of Elastic are you using?" should be filled automatically with the Kibana version
